src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DefaultJudgementHandler.java

```diff
@@ -57,9 +57,10 @@ public class DefaultJudgementHandler extends CallbackHandler {
     public static final String JUDGMENT_REQUESTED = "# Judgment for damages to be decided requested ";
     public static final String JUDGMENT_GRANTED_HEADER = "# Judgment for damages to be decided Granted ";
     private static final List<CaseEvent> EVENTS = List.of(DEFAULT_JUDGEMENT);
+    private static final int DEFAULT_JUDGEMENT_DEADLINE_EXTENSION_MONTHS = 24;
```

Comment:
I think we externalise this. - via flux config?

Reply:
I believe this won't be changed so frequently to justify that,  Given that I think I is best to have it declared in the code so it is visible straight away what the business rule is for each event.